core/spec/models/spree/variant_spec.rb

```diff
@@ -497,4 +497,17 @@
       expect(Spree::Variant.in_stock).to eq [in_stock_variant]
     end
   end
+
+  context '#volume' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.